CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ For more information about changelogs, check
 [Keep a Changelog](http://keepachangelog.com) and
 [Vandamme](http://tech-angels.github.io/vandamme).
 
+## 0.34.0 - 2023-12-22
+
```

Comment:
I personally think that links should be added for each item that's changed. It'll make life easier for the people that read this